common-rest/src/test/java/org/uniprot/api/rest/download/heartbeat/HeartBeatProducerTest.java

```diff
@@ -92,14 +158,4 @@ void stop() {
         assertEquals(70L, afterStopped.getEntriesProcessed());
         assertTrue(afterStopped.getUpdated().isAfter(LocalDateTime.now().minusMinutes(2)));
     }
-
-    @Test
-    void create_whenExceptionsOccurs() {
```

Comment:
We can keep this test

Reply:
it was not actually deleted, just changed the order ;)